### PR TITLE
Removed Authors from domains

### PR DESCRIPTION
**Pull request by Aishubalaji**

### CODE REVIEW

**Comment by Meldiron** [comment]
Linking issue: https://github.com/appwrite/appwrite/issues/4415

File: tests/DomainTest.php [hunk 1]
```diff
@@ -6,7 +6,6 @@
  * @subpackage Tests
  *
  * @link https://github.com/utopia-php/framework
- * @author Eldad Fux <eldad@appwrite.io>
```
Comment (Meldiron) [inline tests/DomainTest.php:9]: Let's remove the whole comment blocks, please.

**Review by Meldiron — CHANGES_REQUESTED** [review]
One tiny comment 🤏 I can mark as approved once resolved.

**Comment by Aishubalaji** [comment]
@Meldiron can you review this?

**Pull request closed without merging**